webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -244,8 +244,8 @@ def bugzilla_submit_url(report, bug_product):
         'product': bug_product,
         'op_sys': op_sys,
         'rep_platform': report['cpu_name'],
-        'cf_crash_signature': '[@ {}]'.format(report['signature']),
-        'short_desc': 'Crash in {}'.format(report['signature']),
+        'cf_crash_signature': u'[@ {}]'.format(report['signature']),
+        'short_desc': u'Crash in {}'.format(report['signature']),
```

Comment:
I looked through the codebase and I'm kind of surprised we don't have something already.

I'm thinking something like Django's `smart_str` (https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.smart_str).

If you want, I can write it and add it to `socorro/lib/` and then you can use that. Would that help?